pygyw/layout/helpers.py

```diff
@@ -2,6 +2,7 @@
 
 from . import settings
 from . import fonts
+from ..color import Color
```

Comment:
Why is color there above ? Shouldn't it be at the layout level ?

Reply:
Forgot to remove the import. It was unused.
Do you want me to move color.py inside layout?

Reply:
Yes, I would prefer that